services/api/src/tasks/index.ts

```diff
@@ -34,6 +36,17 @@ export function configureTasks(app: FastifyInstance) {
     )
   )
 
+  app.scheduler.addSimpleIntervalJob(
+    new SimpleIntervalJob(
+      { minutes: 3 },
+      new AsyncTask(
+        'store-collectibles',
+        async () => await storeCollectiblesTask(app.container),
+        (error) => app.log.error(error)
+      )
+    )
+  )
```

Comment:
When running locally I got the error I started getting `Request failed with status code 429`, so I think we ran into the rate limiting.